### PR TITLE
[NO-TASK] remove spider api & services

### PR DESCRIPTION
**Pull request by Kolibri1990**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations
  - Login/logout
  - Registration
  - Update user data
  - Create/delete payment routes

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

File: src/subdomains/generic/user/models/user-data/user-data.service.ts [hunk 1]
```diff
@@ -410,8 +363,7 @@ export class UserDataService {
     const userDataList = await this.userDataRepo.findBy({ kycFileId: MoreThan(0) });
     const idList = [];
     for (const userData of userDataList) {
-      const customerInfo = await this.spiderApiService.getCustomerInfo(userData.id);
-      if (customerInfo && !customerInfo.contractReference) idList.push(userData.id);
+      idList.push(userData.id);
```
Comment (Yannick1712) [inline src/subdomains/generic/user/models/user-data/user-data.service.ts:366]: define idList with map Method?
`return userDataList.map((u) => u.id)`

**Review by TaprootFreak — APPROVED** [review]

File: src/subdomains/generic/user/models/ident/ident.controller.ts
Comment (davidleomay) [inline src/subdomains/generic/user/models/ident/ident.controller.ts:1]: Can be removed

File: src/subdomains/generic/user/models/user-data/user-data.service.ts (hunk 1 above)
Comment (davidleomay) [reply src/subdomains/generic/user/models/user-data/user-data.service.ts:366]: The method `getAllUserDataWithEmptyFileId` seems to be obsolete. I would remove it.

File: src/subdomains/generic/user/models/user-data/user-data.controller.ts
Comment (davidleomay) [inline src/subdomains/generic/user/models/user-data/user-data.controller.ts:1]: All "TODO-methods" can be removed

File: src/subdomains/generic/user/models/kyc/kyc.controller.ts [hunk 2]
```diff
@@ -91,32 +93,39 @@ export class KycController {
...
   ): Promise<boolean> {
-    return this.kycService.uploadDocument('', files[0], KycDocument.INCORPORATION_CERTIFICATE, jwt.id);
+    const upload = await this.storageService.uploadFile(
+      jwt.id,
+      KycFileType.USER_NOTES,
+      files[0].filename,
+      files[0].buffer,
+      files[0].mimetype as KycContentType,
+    );
+    return upload != '';
```
Comment (davidleomay) [inline src/subdomains/generic/user/models/kyc/kyc.controller.ts:103]: `jwt.id` is not UserData ID

Comment (davidleomay) [inline src/subdomains/generic/user/models/kyc/kyc.controller.ts:103]: Use the following file name:
```ts
`${Util.isoDate(new Date()).split('-').join('')}-incorporation-certificate-${filename}`
```

File: src/subdomains/generic/user/models/kyc/kyc.controller.ts [hunk 3]
```diff
@@ -132,29 +141,27 @@ export class KycController {
     @Param('code') code: string,
     @UploadedFiles() files: Express.Multer.File[],
   ): Promise<boolean> {
-    return this.kycService.uploadDocument(code, files[0], KycDocument.INCORPORATION_CERTIFICATE);
+    return; //TODO
```
Comment (davidleomay) [inline src/subdomains/generic/user/models/kyc/kyc.controller.ts:144]: Use new storage service

File: src/subdomains/generic/user/models/kyc/kyc.controller.ts
Comment (davidleomay) [inline src/subdomains/generic/user/models/kyc/kyc.controller.ts:1]: I think we need to revert all TODO-methods.

**Review by davidleomay — APPROVED** [review]

**Pull request merged**